hooks/lib/android/manifestWriter.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
  * @param {Object} pluginPreferences - plugin preferences as JSON object; already parsed
  */
 function writePreferences(cordovaContext, pluginPreferences) {
-  var pathToManifest = path.join(cordovaContext.opts.projectRoot, 'platforms', 'android', 'AndroidManifest.xml');
+  var pathToManifest = path.join(cordovaContext.opts.projectRoot, 'platforms', 'android', 'app', 'src', 'main', 'AndroidManifest.xml');
```

Comment:
This change it not backwards compatible and will break build with older Cordova android versions.  Maybe you should check the cordova android version or copy the whole manifest file so it is compatible with older project structures.